astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -974,52 +981,56 @@ def test_data_out_of_range(parallel, reader):
     """
     # Python reader and strtod() are expected to return precise results
     rtol = 1.e-30
-    if reader > 1:
-        rtol = 1.e-15
-    # passing fast_reader dict with parametrize does not work!
-    if reader > 0:
-        fast_reader = {'parallel': parallel, 'use_fast_converter': reader > 1}
-    else:
-        fast_reader = False
+
+    # update fast_reader dict
+    if fast_reader:
+        fast_reader['parallel'] = parallel
+        if fast_reader.get('use_fast_converter'):
+            rtol = 1.e-15
+
     if parallel:
-        if reader < 1:
+        if not fast_reader:
             pytest.skip("Multiprocessing only available in fast reader")
         elif TRAVIS:
             pytest.xfail("Multiprocessing can sometimes fail on Travis CI")
 
-    fields = ['10.1E+199', '3.14e+313', '2048e+306', '0.6E-325', '-2.e345']
-    values = np.array([1.01e200, np.inf, np.inf, 0.0, -np.inf])
-    t = ascii.read(StringIO(' '.join(fields)), format='no_header', guess=False,
-                   fast_reader=fast_reader)
+    fields = [ '10.1E+199', '3.14e+313', '2048e+306', '0.6E-325', '-2.e345' ]
```

Comment:
The extra space after `[` and before `]` is not PEP-8 compliant, please revert all these changes.  In general it is good to strive for minimal diffs, especially non-impacting diffs.  This makes life easier for reviewers and maintains the style of the original code.

Reply:
Sorry, I had those lists without the spaces in my local repo (or actually still have on the machine I am working on right now...) - must have gone bad during rebasing, weird.